README.md

```diff
@@ -139,17 +139,51 @@ Note that Ray Client uses the [ray://](https://docs.ray.io/en/master/cluster/ray
 
 When using the `RayTaskRunner` with a remote Ray cluster, you may run into issues that are not seen when using a local Ray instance. To resolve these issues, we recommend taking the following steps when working with a remote Ray cluster:
 
-1. If your task fails with permission or file not found errors, set `PREFECT_LOCAL_STORAGE_PATH` in your Prefect settings to a path accessible on both the remote cluster and the machine executing the flow:
-```bash
-prefect config set PREFECT_LOCAL_STORAGE_PATH='/tmp/prefect/storage'
+1. By default, Prefect will not persist any data to the filesystem of the remote ray worker. However, if you want to take advantage of Prefect's caching ability, you will need to configure a remote result storage and persist the results, like shown below:
+```python
+from typing import List
+
+from prefect import flow, get_run_logger, task
+from prefect.filesystems import S3
+from prefect.tasks import task_input_hash
+from prefect_ray.task_runners import RayTaskRunner
+
+
+@task(cache_key_fn=task_input_hash)
+def say_hello(name: str) -> None:
+    logger = get_run_logger()
+    # this should only show if not cached
+    logger.info(f"hello {name}!")
+    return name
+
+
+@flow(
+    task_runner=RayTaskRunner(
+        address="ray://<instance_public_ip_address>:10001",
+    ),
+    persist_result=True,
+    result_storage=S3(bucket_path="<bucket_path>"),
```

Comment:
I think we should encourage people to create their storage via the UI.
```suggestion
    # Using an S3 block that has already been created via the Prefect UI
    result_storage=S3.load("my_results_storage"),
```

Reply:
I'd use a slug there so it's loaded at runtime instead of script execution, e.g. `s3/my_result_storage`